collections/guides/encrypt_us_bank_accounts/index.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: default
+title: Encrypt U.S. Banks in your applications
+
+permalink: /guides/encrypt-us-banks-in-your-applications/
+categories: guides
+nav_order: 3
+has_children: true
+has_toc: false
+description: Basis Theory
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+# Encrypt U.S Banks in your applications
+{: .no_toc }
+
+<span class="base-alert success">
+  <span>
+    This guide will quickly enable you to be compliant with new Nacha encryption rules. 
+  </span>
+</span>
+
+Your customer's bank information is as sensitive as their credit card data, and needs to be stored with the same care. Encrypting this data takes a strong encryption pattern, including a KMS, key rotation, multiple encryption keys, access controls, and so much more. 
+
+In this guide, we will show you how to take an existing API and use Basis Theory to safely store the bank data while and retain the exact application functionality. 
+
+If you'd like to follow along with this guide jump right into the code - <a href="https://github.com/Basis-Theory/basis-theory-js-examples/store-atomic-banks">Find it here!</a>
+
+### Table of contents
+{: .no_toc .text-delta }
+1. 
+{:toc}
+
+## The existing application
+
+Below is an existing nodejs application that is currently storing bank data in memory, this will be used as our foundation as we update to more securely store the bank data. 

```

Comment:
```suggestion
Below is an existing Node.js application that is currently storing bank data in memory, this will be used as our foundation as we update to more securely store the bank data. 

```